src/app/pages/MapPage/components/DrawTool/DrawTool.tsx

```diff
@@ -148,6 +150,9 @@ const DrawTool: FunctionComponent = () => {
     const distanceText = getDistanceLabel(layer)
     getDrawingData(layer, distanceText)
     bindDistanceAndAreaToTooltip(layer, distanceText)
+    if (layer.getElement()) {
+      L.DomUtil.addClass(layer.getElement() as HTMLElement, DRAWN_ITEM_CLASS)
+    }
```

Comment:
Instead of typecasing to `HTMLElement` and calling `getElement()` twice, you can instead use `instanceof` to discriminate the type.

```suggestion
    const element = layer.getElement()

    if (element instanceof HTMLElement) {
      L.DomUtil.addClass(element, DRAWN_ITEM_CLASS)
    }
```